src/game/client/terrainroads.h

```diff
@@ -54,6 +54,15 @@ class TerrainRoadType : public MemoryPoolObject
     float Get_Road_Width_In_Texture() { return m_roadWidthInTexture; }
     float Get_Road_Width() { return m_roadWidth; }
 
+    float Get_Bridge_Scale() { return m_bridgeScale; }
```

Comment:
These are names not models or textures so the function name should reflect that

Reply:
I checked and as of now I am using exactly the names found in zh mac, wbi and rts.map